TIMES-NZ-INTERNAL-QA/src/times_nz_internal_qa/app/helpers/data_processing.py

```
@@ -187,7 +187,7 @@ def make_chart_data(
     )
 
     # identify unit
-    unit = unit_list[0]
+    unit = unit_list[0] if unit_list else None
```

If there is no unit_list, it would probably be better for this to fail loudly, since all our inputs and charts should have units